project_script/main.py

```diff
@@ -104,8 +102,49 @@ def integrate_data():
 
 density_of_carriers = [element * CHARGE_DENSITY_CALCULATION for element in integration_results]
 
-pd.DataFrame(density_of_carriers).to_csv(
-    "C:/Users/robee/Desktop/density_of_carriers_test.txt",
-    sep='\t',
-    decimal=','
-    )
+# pd.DataFrame(density_of_carriers).to_csv(
+#     "C:/Users/robee/Desktop/density_of_carriers_test.txt",
+#     sep='\t',
+#     decimal=',',
+#     )
+
+odd_columns_subtracted_smoothed_transposed = np.array(dt.smooth(odd_columns_subtracted_transposed_in_arrays))
+
+peaks = dt.find_the_peaks(odd_columns_subtracted_smoothed_transposed)
```

Comment:
Peaks of what? graphic_peaks, potential_peaks or something like that, would be more readable.